tools/mo/openvino/tools/mo/front/tf/MapFNTransformation.py

```diff
@@ -5,15 +5,15 @@
 
 import numpy as np
 
-from openvino.tools.mo.front.tf.WhileNormalize import WhileNormalize
-from openvino.tools.mo.ops.loop import Loop
 from openvino.tools.mo.front.common.partial_infer.utils import int64_array, mo_array
 from openvino.tools.mo.front.common.replacement import FrontReplacementSubgraph
+from openvino.tools.mo.front.tf.WhileNormalize import WhileNormalize
 from openvino.tools.mo.front.tf.custom_subgraph_call import skip_nodes_by_condition
 from openvino.tools.mo.front.tf.graph_utils import create_op_with_const_inputs
 from openvino.tools.mo.graph.graph import Graph, Node, rename_nodes
 from openvino.tools.mo.middle.pattern_match import find_pattern_matches, inverse_dict
 from openvino.tools.mo.ops.const import Const
+from openvino.tools.mo.ops.loop import Loop
 from openvino.tools.mo.ops.squeeze import Squeeze
 from openvino.tools.mo.ops.unsqueeze import Unsqueeze
 
```

Comment:
please update `ObjectDetectionAPITransformationsFinish` from `ObjectDetectionAPI.py` to run it before new transformations

Reply:
Done.